### PR TITLE
FrontEnd

### PR DESCRIPTION
**Pull request by adliebster**
I updated the note.ts file in order to include a unique ID that is created using a UUID. Also in the reusable profile componenet.ts file, I have added in a function that allows us to delete notes. In addition, I changed the typecast in program.ts as we discussed one afternoon.

### CODE REVIEW

File: src/app/reuseable-components/profile/profile.component.html [hunk 1]
```diff
@@ -35,5 +35,9 @@ <h1>This is where errors will be displayed:</h1>
     <p>Note content: {{note._note}}</p>
     <p>Note author: {{note._userId}}</p>
     <p>Note date: {{note._creationDate}}</p>
+    <p>Note ID: {{note._note_id}}</p>
```
Comment (amarshall45676) [inline src/app/reuseable-components/profile/profile.component.html]: Dont display the noteID

**Review by amarshall45676 — CHANGES_REQUESTED** [review]
Looks good besides the one comments. Also make sure not to commit the pack-lock.json file. You can remove it by doing something like this: 
git rm FILENAME
git commit -m "remove FILE NAME"

**Comment by adliebster** [comment]
I removed the noteID from the display and checked to make sure that it worked. I then while committing did not add in the package-lock.json file.

**Comment by amarshall45676** [comment]
LGTM

**Pull request merged**